webpack.config.js

```diff
@@ -5,6 +5,11 @@ var definePlugin = new webpack.DefinePlugin({
     __DEBUG__: JSON.stringify(process.env.NODE_ENV === 'development'),
 });
 
+var providePlugin = new webpack.ProvidePlugin({
```

Comment:
I don't know. I haven't touched Promise and fetch because using itowns without either of them is quite hard, so it made sense to me to expose them in window. For TextEncoder and TextDecoder, it's only used internally so I didn't want to expose them needlessly.